anchore_engine/analyzers/syft/__init__.py

```diff
@@ -19,11 +21,6 @@ def filter_fn(relationship):
 
 def filter_artifacts(artifacts, relationships):
     def filter_fn(artifact):
-        # syft may do more work than what is supported in engine, ensure we only include artifacts
-        # of select package types.
-        if artifact["type"] not in modules_by_artifact_type:
```

Comment:
Does any of the upstream logic that calls this function need this check?

Reply:
nothing else calls the function

Reply:
Ack, thanks Vijay